assignments/arrays.js

```diff
@@ -63,32 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car33 = inventory[32]
```

Comment:
this works for now, but what happens when a car is added or removed from the inventory? or somehow the order of the array gets shuffled and id:33 is now in array position 6?
your code would be more robust if it combed through the inventory and selected the car based on the id property on each object.
the find method is a perfect way to do this

```
const car33 = inventory.find(car => car.id === 33);
```